authservice/app/usecase/login_uc.go

```diff
@@ -0,0 +1,59 @@
+package usecase
+
+import (
+	"AdsService/authservice/app/dto"
+	"AdsService/authservice/app/uc_errors"
+	"AdsService/authservice/domain/entity"
+	"AdsService/authservice/domain/port"
+	"golang.org/x/crypto/bcrypt"
+	"time"
+)
+
+type LoginUC struct {
+	Users    port.UserRepository
+	Sessions port.SessionRepository
+	Tokens   port.TokenRepository
+}
+
+func (uc *LoginUC) Execute(in dto.LoginDTO) (dto.AuthResponseDTO, error) {
```

Comment:
```suggestion
func (uc *LoginUC) Execute(in dto.Login) (dto.AuthResponseDTO, error) {
```
пакте и так dto в имени можно везде опустить